Userland/Libraries/LibJS/Runtime/DatePrototype.cpp

```diff
@@ -967,7 +967,7 @@ JS_DEFINE_NATIVE_FUNCTION(DatePrototype::to_iso_string)
     if (!Value(this_object->date_value()).is_finite_number())
         return vm.throw_completion<RangeError>(ErrorType::InvalidTimeValue);
 
-    auto string = this_object->iso_date_string();
+    auto string = this_object->iso_date_string().release_value_but_fixme_should_propagate_errors();
```

Comment:
Is there any reason that this can't use `TRY`?

Reply:
Hmm, I get a weird template error like so - "no matching constructor for initialization of 'JS::Value'" when I replace it with ```TRY(this_object->iso_date_string())```. Sry, I am new to this code-base and am still figuring out the right error-propagation patterns.

Reply:
Got it. Thank You. Changed it to ```TRY_OR_THROW_OOM``` for now. Will look more into the patterns.  The function itself returns a ```ThrowCompletionOr<String>```.